src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java

```diff
@@ -195,6 +204,21 @@ public void run() {
                         }
                     }
 
+                    final Set<String> deprecatedAuditKeysInSettings = AuditConfig.getDeprecatedKeys(settings);
+                    if (!deprecatedAuditKeysInSettings.isEmpty()) {
+                        LOGGER.warn("Following keys {} are deprecated in elasticsearch settings. They will be removed in plugin v2.0.0.0", deprecatedAuditKeysInSettings);
+                    }
+                    final boolean isAuditConfigDocPresentInIndex = cl.isAuditConfigDocPresentInIndex();
+                    if (isAuditConfigDocPresentInIndex) {
+                        if (!deprecatedAuditKeysInSettings.isEmpty()) {
```

Comment:
What is the final agreement on the case where audit entries are present both in the index and elasticsearch.yml?

Reply:
Warn the user that settings are present at both places but continue using from index. Do not fail startup.  
Index takes priority over elasticsearch.yml.

Reply:
OK, just want to be sure that it is the final agreement and it is documented